Gemfile

```
@@ -8,6 +8,7 @@ gem 'brakeman'
 gem 'bundler-audit'
 gem 'byebug'
 gem 'flay'
+gem 'nokogiri', '1.16.2'
```

Maybe match the same requirement as the gemrc.
```suggestion
gem 'nokogiri', '~> 1.16.2'
```